packages/language-server/src/commads.ts

```diff
@@ -17,7 +16,7 @@ export function executeCommand(
   }
 
   switch (params.command) {
-    case QUICK_FIX_STABLE_ID_COMMAND: {
+    case quickFixCommands.STABLE_ID_ERROR.command: {
```

Comment:
the `command` property name seems not 100% clear to me here.
should/could we give this property another name?